scripts/Linux-Pkg/Makefile

```diff
@@ -49,7 +49,15 @@ build: pack-source
 	rm -rf Output/pkg
 	rm -rf Output/src
 
+aur:
+	mkdir -p Output
+	git clone ssh://aur@aur.archlinux.org/kathara Output/kathara-aur
+	mv kathara-aur/PKGBUILD pacman/kathara.install Output/kathara-aur
```

Comment:
Better `cp` this one instead of `mv`, so we don't lose the original file

Reply:
You're right :smile: